_posts/en/newsletters/2019-04-02-newsletter.md

```diff
@@ -0,0 +1,269 @@
+  using `OP_CHECKLOCKTIMEVERIFY` (CLTV).  Bob then sends the money
+  (minus his routing fee) to Charlie with similar terms except that he
+  subtracts the value of Charlie's CLTV delta from Alice's original 400
+  blocks, reducing the value to 360 blocks and ensuring that nobody else
+  can cheat Charlie out of having at least 40 blocks to claim his money
+  onchain if Bob becomes uncooperative despite the payment succeeding.
+  Subsequent routers each successively subtract their delta from the
+  value of the terms they give until the final receiver (Zed) might only
+  have a few blocks in which to either claim the payment or lose his
+  chance to receive the money (causing the whole series of uncompleted
+  payments to unwind).  According to the git commit messages, this
+  change is being made because "the initial value was purposefully very
+  high.  [...] Such a high value discourages path finding algorithms
+  from taking LND paths due to the negative impact of CLTV value in
+  widely deployed heuristics."
+
+- [Eclair #894][] replaces the JSON-RPC interface with an HTTP
+  POST interface.  Instead of RPC commands, HTTP endpoints are used
+  (e.g. the `channelstats` RPC is now `POST
+  http://localhost:8080/channelstats`).  Parameters are provided to the
+  endpoint using named form parameters with the same JSON syntax as
+  used with RPC parameters.  Returned results are identical to before
+  the change.  The old interface is still available using the
+  configuration parameter `eclair.api.use-old-api=true`, but it is
+  expected to be removed in a subsequent release.  See the [updated API
+  documentation][eclair api docs] for details.
+
+{% include references.md %}
+{% include linkers/issues.md issues="15555,15560,15620,15643,15519,15637,2759,894,13044" %}
+[0.18.0]: https://bitcoincore.org/bin/bitcoin-core-0.18.0/
+[bech32 easy]: {{news38}}#bech32-sending-support
+[poly1305]: https://en.wikipedia.org/wiki/Poly1305
+[0.16.0 segwit]: https://bitcoincore.org/en/releases/0.16.0/#segwit-wallet
+[eclair api docs]: https://acinq.github.io/eclair/#introduction
+[johoe's mempool statistics]: https://jochen-hoenicke.de/queue/#0,1w
+[p2sh.info's fee estimate tracker]: https://p2sh.info/dashboard/db/fee-estimation?orgId=1
+[trampoline thread]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-March/001939.html
+[core meeting segwit]: http://www.erisian.com.au/meetbot/bitcoin-core-dev/2019/bitcoin-core-dev.2019-03-28-19.01.log.html#l-83
+[bech32 adoption]: https://en.bitcoin.it/wiki/Bech32_adoption
+[bech32 reference libraries]: https://github.com/sipa/bech32/tree/master/ref
+[segwit addr to bytes]: https://github.com/sipa/bech32/blob/master/ref/python/tests.py#L30
+[bitgo segwit]: https://blog.bitgo.com/native-segwit-addresses-via-bitgos-api-4946f2007be9
+[gemini reddit]: https://old.reddit.com/r/Bitcoin/comments/b66n0v/psa_gemini_is_full_on_with_native_segwit_and_uses/
```

Comment:
Should we be linking to "old" Reddit? ie is there more danger this link breaks in the future than using www.reddit.com?